helm/values.yaml

```diff
@@ -106,6 +102,20 @@ entityServiceConfig:
     host: attribute-service
     port: 9012
 
+database:
```

Comment:
@suresh-prakash This impacts how datastore type is configured on on-prem platform. Please keep that in mind when making such changes.

CC: @ravisingal  @kotharironak